locales/en.yml

```
@@ -98,6 +98,7 @@ channel-ab-testing: Enable AB testing
 channel-ab-testing-percentage: Percentage of users recving secondary version
 channel-create: Create channel
 channel-deleted: Channel deleted
+channel-forced-devices: Channel-Forced Devices
```

Shouldn't we use 
```suggestion
channel-forced-devices: Channel-forced devices
```
instead?

In `channel-deleted` and `channel-created` for example, we don't have capital letters on all words, so would be better to have it in the same way for consistency